APE13.rst

```diff
@@ -0,0 +1,409 @@
+                                   {‘frame’: ‘icrs’}),
+                          ‘spec’: (‘astropy.units.Wavelength`, {}),
+                          ‘time’: (‘astropy.time.Time’,
+                                   {‘format’:’mjd’, ‘scale’:’utc’})}
+
+**The identity transform** for a 1D array (i.e., pixel -> pixel):
+
+.. code-block:: python
+
+    wcs.axis_correlation_matrix = [[True]]
+    wcs.world_axis_units = [‘pixel’]
+    wcs.world_axis_physical_type = [‘instr.pixel’]
+    wcs.world_axis_object_components = [(‘spec’, 0)]
+    wcs.world_axis_object_classes  = {‘spec’:(‘astropy.units.pixel’: {})}
+
+Common UCD1+ names
+^^^^^^^^^^^^^^^^^^
+
+The full UCD1+ vocabulary (used for the ``world_physical_types`` attribute)
+includes a large number of options, but here we summarize some of the most
+common ones that will likely be used::
+
+    em.energy                | Energy value in the em frame
+    em.freq                  | Frequency value in the em frame
+    em.wavenumber            | Wavenumber value in the em frame
+    em.wl                    | Wavelength value in the em frame
+    instr.pixel              | Pixel (default size: angular)
+    pos.az.alt               | Alt-azimutal altitude
+    pos.az.azi               | Alt-azimutal azimut
+    pos.cartesian.x          | Cartesian coordinate along the x-axis
+    pos.cartesian.y          | Cartesian coordinate along the y-axis
+    pos.cartesian.z          | Cartesian coordinate along the z-axis
+    pos.ecliptic.lat         | Ecliptic latitude
+    pos.ecliptic.lon         | Ecliptic longitude
+    pos.eq.dec               | Declination in equatorial coordinates
+    pos.eq.ra                | Right ascension in equatorial coordinates
+    pos.galactic.lat         | Latitude in galactic coordinates
+    pos.galactic.lon         | Longitude in galactic coordinates
+    pos.healpix              | Hierarchical Equal Area IsoLatitude Pixelization
+    spect.dopplerVeloc       | Radial velocity, derived from the shift of some spectral feature
+    spect.dopplerVeloc.opt   | Radial velocity derived from a wavelength shift using the optical convention
+    spect.dopplerVeloc.radio | Radial velocity derived from a frequency shift using the radio convention
+    time                     | Time, generic quantity in units of time or date
+    time.epoch               | Instant of time related to a generic event (epoch, date, Julian date, time stamp/tag,...)
+
+High-level Astropy Object
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Unlike the low-level API, the ‘high-level’ interface described here will be a
+single Astropy-developed class since it interfaces with various Astropy objects.
+This high-level API would provide the ability for example to get ``SkyCoord``,
+``Time`` etc. objects back from a pixel to world conversion, and conversely to
+be able to convert ``SkyCoord``, ``Time`` etc. to pixel values.
+
+The high-level object would not inherit from the low-level classes but instead
```

Comment:
Should the high-level object drop the ``pixel_to_world_values`` and ``world_to_pixel_values`` methods?

Reply:
The high-level object won't have the low-level methods - instead these will be available as ``w.low_level_wcs. pixel_to_world_values`` for example (see below for a mention of ``pixel_to_world_values``

Reply:
Ok, so that's not how I read this section. Is this section then saying the only methods required on high level are `world_to_pixel` and `pixel_to_world`?

Reply:
Yes - bear in mind the high level object is not an API specification but an actual object. We could definitely add more to it over time if we want. This is just the minimum.

Reply:
ok, I would say that needs clarifying a little.

Reply:
@Cadair - can you check my latest changes in e31fd0f9d63f71a4f8e0281a978f4e11309d0d3a

Reply:
looks good